docs/content/documentation/templates/overview.md

```diff
@@ -212,10 +212,20 @@ As a security precaution, if this file is outside the main site directory, your
 {% set data = load_data(path="content/blog/story/data.toml") %}
 ```
 
+The optional `required` boolean argument can be set to false so that missing data does not produce an error, but returns a null value instead. For URLs, all HTTP errors are discarded. For local paths, missing file errors are discarded, but permissions errors are kept. In both cases, invalid data that could not be parsed to the requested data format will still produce an error.
+
+The snippet below outputs the HTML from a Wikipedia page, or "No data found" if the page was not reachable, or did not return a successful HTTP code:
+
+```jinja2
+{% set data = load_data(path="https://en.wikipedia.org/wiki/Commune_of_Paris", required=false %}
+{% if data %}{{ data | safe }}{% else %}No data found{% endif %}
```

Comment:
I believe you need to use `{%/*` and `*/%}` in that snippet to avoid zola calling it.

Reply:
This does not seem to be the case (anymore?). i rendered the docs and it appeared as expected.

For example markdown filter is already described like this:

````
```jinja2
{{ some_text | markdown(inline=true) }}
```
````

Reply:
Hu weird